quiz.html

```diff
@@ -11,33 +11,29 @@
 		</header>
 		
 		<div class="wrapper">
+			<div class="quiz-page">
+				<h1>Skin Type Quiz</h1>
 
-			<!-- This div is for the skin type quiz and is different to the other pages -->
-			<div class="quizbox">
-				<h1>Question 1 heading</h1>
-				<h2>1 / 5</h2>
-				<p>Question related images and text will go here</p>
-				<hr color="white" width="80%" size="3">
+				<!-- This div is for the skin type quiz and is different to the other pages -->
+				<div class="quizbox">

```

Comment:
This version contains only 1 question per page and requires the user to click the "next" or "previous" button.